ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/as-path.j2

```diff
@@ -0,0 +1,19 @@
+{# eos - as path #}
+{% if as_path is arista.avd.defined %}
+!
+{%     if as_path.regex_mode is arista.avd.defined %}
+ip as-path regex-mode {{ as_path.regex_mode }}
+{%     endif %}
+{%     if as_path.access_lists is arista.avd.defined %}
+{%         for as_path_access_list in as_path.access_lists %}
```

Comment:
Use ```natural_sort``` filter. Same for all for loops in this template.

Reply:
Thanks Tony, let me add the filter. I think I'll make use of it by default in the future.

Reply:
You should keep the changes to the outer for-loop. so add the `natural_sort` and remove the check in line 7. Same for documentation. It was only the `entries` which should _not_ use the sorting, since it will fail.

There is another trick that you can use instead of natural_sort, if you don't want to sort, but still want to include the variable check on the `for` loop.
```jinja
{% for entry in entries | arista.avd.default([]) %}
```

But what you have now is fine, since you also need to check for name.

Reply:
just to make sure I understand 100% correctly: the whole point is to include the check if as_path_access_list.name is defined in the for loop, right? so basically safe an explicit if-condition for it?